modules/runner/src/integrationTest/sql/filter/test_obsolete_filters.test_ignore

```diff
@@ -1,281 +0,0 @@
-# name: test/sql/filter/test_obsolete_filters.test
-# description: Test expressions with obsolete filters
-# group: [filter]
-# Ignored https://issues.apache.org/jira/browse/IGNITE-15561
```

Comment:
the same - need to not forget to close the issue

Reply:
sure